### PR TITLE
Cluster Grenade Cleanup

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Update cluster grenades to be in line with TG.

Also adds in some extra badmin grenades...because I felt like it and wanted to make them.

- More logically oriented, less duplicate code, and segments don't runtime.
- restructures where clusterbusters are located
- Adds in a few more badmin clusterbuster grenades
 - Drugs/Hippie
 - Fox
 - Hellwater
 - Tabun
 - Crab
 - Random Toys
 - Random Tools
 - Random primary tool storage stuff
 - Evil Banana grenade (the one with the damaging banana peels)
 - Cluster grenades of cluster grenades
- moves some special/custom chem grenades into their own file under the proper folder
- Adds in a few new badmin chem grenade: hellwater grenade
- Fixes incendiary grenades not working
 - removes the now unused incendiary fuel reagent
- fixes a container having a non-existent reagent in it.

### CODE REVIEW

File: code/game/objects/items/weapons/grenades.dm [hunk 1]
```diff
@@ -292,64 +281,145 @@
...
+
+	qdel(src)
+
+//////////////////////////////////
+//The payload spawner effect
+/////////////////////////////////
+/obj/effect/payload_spawner/New(var/turf/newloc,var/type, var/numspawned as num)
+
+	for(var/loop = numspawned ,loop > 0, loop--)
+		var/obj/item/weapon/grenade/P = new type(loc)
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/grenades.dm]: This only working for grenades seems to be entirely arbitrary, all it has to do is run an istype() for grenades after the walk_away proc.

**Comment by taukausanake** [comment]
What is the fluffy love bomb?

File: code/game/objects/items/weapons/grenades.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/grenades.dm]: It would have to run it before that, too, @tigercat2000 

Not many things have an active var =p

Comment (tigercat2000) [inline code/game/objects/items/weapons/grenades.dm]: this entire loop should look like this

```
for(var/loop = numspawned, loop > 0, loop--)
    var/atom/movable/P = new type(loc)
    if(!istype(P)) return
    walk_away(P, loc, rand(1, 4))
    if(istype(P, /obj/item/weapon/grenade))
        P.active = 1
        spawn(rand(15, 60))
            if(P && isnull(P.gcDestroyed))
                P.prime()
            qdel(src)
     else qdel(src)

**Comment by Fox-McCloud** [comment]
Added in the old grenades and changed the code to support non-grenade payloads without runtiming.

File: code/game/objects/items/weapons/grenades.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/grenades.dm]: Yup. That's exactly what I did =p

**Comment by Fox-McCloud** [comment]
@Markolie and @TheDZD 

Ready.

**Pull request merged**